.github/workflows/macos.yml

```diff
@@ -39,7 +39,7 @@ jobs:
         mkdir build
         cd build
         : # See https://github.com/OSGeo/gdal/issues/6949 (Java bindings) and https://github.com/OSGeo/gdal/pull/8226 (Arrow and Parquet)
-        cmake -DCMAKE_INSTALL_PREFIX=$GDAL_INSTALL_PATH -DCMAKE_BUILD_TYPE=Release -DBUILD_JAVA_BINDINGS=ON -DBUILD_PYTHON_BINDINGS=OFF ${CMAKE_OPTIONS} ..  
+        cmake -DCMAKE_INSTALL_PREFIX=$GDAL_INSTALL_PATH -DCMAKE_BUILD_TYPE=Release -DBUILD_JAVA_BINDINGS=ON -DBUILD_PYTHON_BINDINGS=OFF -DGDAL_USE_POPPLER=OFF ${CMAKE_OPTIONS} ..
```

Comment:
This seems clear, any reason not to go ahead?

Reply:
PRs needs to be reviewed, that's all. We don't merge PRs without review, unless there is some urgency.